src/sgame/sg_bot.cpp

```diff
@@ -205,7 +205,28 @@ bool G_BotSetDefaults( int clientNum, team_t team, int skill, const char* behavi
 			return false;
 		}
 	}
+	return true;
```

Comment:
If you are changing the behavior of an existing bot, don't you need to do the stuff with clearing the current/running nodes?

Reply:
I don't know. It seems to work without, but you are probably right. 😄